lib/modules/dosomething/dosomething_campaign/includes/Campaign.php

```diff
@@ -408,6 +413,17 @@ protected function getLanguage() {
   }
 
   /**
+   * Get campaign's latest news.
+   *
+   * @return array
+   */
+  protected function getLatestNews() {
+    return [
+      'latest_news' => dosomething_helpers_extract_field_data($this->node->field_latest_news_copy),
```

Comment:
It returns as an empty array within the `node` so was suspecting maybe it could be more items?